src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -573,7 +582,7 @@ int getPlaceholderSlot(BoundExpression expr)
                 return -1;
             }
 
-            MethodSymbol getTopLevelMethod(MethodSymbol method)
+            static MethodSymbol getTopLevelMethod(MethodSymbol method)
```

Comment:
>static [](start = 12, length = 6)

:memo: I checked and there is already a feature (in 16.1) that will allow us to fix all such local functions. So this will become easier once we start dogfooding 16.1.
[https://github.com/dotnet/roslyn/pull/32941](https://github.com/dotnet/roslyn/pull/32941)